src/overlays/actors/ovl_Obj_Etcetera/z_obj_etcetera.h

```diff
@@ -3,13 +3,32 @@
 
 #include "global.h"
 
+#define OBJETCETERA_TYPE(thisx) (((thisx)->params & 0xFF80) >> 7)
+
+typedef enum {
+    /*  0   */ TYPE_PINK_FLOWER,
+    /*  1   */ TYPE_PINK_FLOWER_SPAWNED_FROM_MAD_SCRUB,
+    /*  2   */ TYPE_GOLD_FLOWER,
+    /*  3   */ TYPE_GOLD_FLOWER_SPAWNED_FROM_MAD_SCRUB,
+} ObjEtceteraType;
+
+#define NUMBER_OF_FLOWER_TYPES 4
+
 struct ObjEtcetera;
 
 typedef void (*ObjEtceteraActionFunc)(struct ObjEtcetera*, GlobalContext*);
 
 typedef struct ObjEtcetera {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x13C];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ SkelAnime skelAnime;
+    /* 0x01A0 */ ColliderCylinder collider;
+    /* 0x01EC */ Vec3s jointTable[11];
+    /* 0x022E */ Vec3s morphTable[11];
+    /* 0x0270 */ f32 intenseOscillationScale;
+    /* 0x0274 */ s16 oscillationTimer;
+    /* 0x0276 */ u16 burrowFlag;
+    /* 0x0278 */ s8 objIndex;
+    /* 0x027C */ Gfx* displayListPtr;
```

Comment:
I think `dList` is simpler
```suggestion
    /* 0x027C */ Gfx* dList;
```

Reply:
I grabbed this from z_bg_iknv_obj.c, but sure, that's simpler.